core/src/main/scala/scala/pickling/pickler/Any.scala

```diff
@@ -22,7 +22,7 @@ object AnyUnpickler extends Unpickler[Any] {
     val actualUnpickler = internal.currentRuntime.picklers.genUnpickler(scala.reflect.runtime.currentMirror, tag)
     actualUnpickler.unpickle(tag, reader)
   }
-  def tag: FastTypeTag[Any] = FastTypeTag[Any]
+  def tag: FastTypeTag[Any] = FastTypeTag.Any
```

Comment:
yeah, it should be.  In this instance, not a huge deal since it's returning the static.